lib/tasks/organisations.rake

```diff
@@ -11,6 +11,6 @@ task download_user_organisations: :environment do
   next unless PullRequest.in_date_range?
   User.all.find_each do |user|
     puts "Importing organisations for #{user.nickname}"
-    user.download_user_organisations(User.load_user.token) rescue nil
```

Comment:
Because the line `   o.login.match(/^coderwall-/)` is removed from `pull_request_downloader.rb`, the function `download_user_organisations` can no longer return nil, thus eliminating the need for the rescue.